### PR TITLE
Domain registration

### PR DESCRIPTION
**Pull request by MrBlockman**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] The file's name is all lowercased and alphanumeric. 
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.  

## Website Link/Preview

[mrblockman.github.io](url)

### CODE REVIEW

File: domains/domains.json
Comment (DEV-DIBSTER) [inline domains/domains.json:1]: Think you forgot to get the actual subdomain you want.

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

File: domains/domains.json
Comment (MrBlockman) [reply domains/domains.json:1]: whoops lol.  should be fixed now

Comment (DEV-DIBSTER) [reply domains/domains.json:1]: File name still is unchanged btw, seems like you created another file that isn't in the domains directory folder. Could you double check?

File: mrblockman.github.io
Comment (DEV-DIBSTER) [inline mrblockman.github.io:1]: Yeah this entire file needs to go, not in the proper directory.

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

**Pull request closed without merging**